### ISSUE TITLE
Support html in the tutorial steps

### ISSUE THREAD

**Issue by offtherailz** [geosolutions-it/MapStore2#1498]
We should support to add html to the tutorial steps, to allow adding images, titles and so on.
This is already available editing the preset. 
This should be supported : 
 - [ ] in `rowStep`, for customization
 - [ ] we need support for i18n for HTML

The target is to provide a fully customizable and internationalizable tutorial

**Comment by allyoucanmap** [comment]
I just tested the current version of the tutorial plugin and we can add react component or dom element in the title and text attributes in the preset files.

``` js
...
{
  title: (<Panel header={'bootstrap panel'}>{'panel content'}</Panel>),
  text: '<img width="100%" src="http://path/img/logo.png"/>',
  selector: '#drawer-menu-button'
},
...
```
![000](https://cloud.githubusercontent.com/assets/19175505/23312966/f545c8a8-fabb-11e6-9e33-297c7b6c5272.png)

**Comment by offtherailz** [comment]
updated the issue desription to specify this

**Comment by offtherailz** [comment]
Fixed  by eccbee0a36a30a4209533eae1092e1bec32a8e61
Going to close
